plugins/tinymcespellchecker.md

```diff
@@ -112,85 +117,99 @@ tinymce.init({
 
 {% include configuration/languages.md %}
 
-### `spellchecker_dialog`
+### `spellchecker_languages`
 
-This option specifies the primary spell checking mode. 
+This option specifies the spellchecker languages that are available to the user, provided as a comma delimited string. The specified languages will be available for the spellchecker when using Spell Checker Pro.
 
-- When set to `true`, the [`spellcheck` toolbar button](#toolbarbuttons) will open a dialog that will step the user through each potential spelling error in the document. 
-- When set to `false`, the [`spellcheck` toolbar button](#toolbarbuttons) will enable or disable as-you-type spell checking and the [`spellcheckdialog` toolbar button](#toolbarbuttons) will open the spell checker dialog.
+**Type:** comma-separated `String`
+
+**Default Value:**
+````
+'US English=en_us,UK English=en_gb,Danish=da,Dutch=nl,Finnish=fi,French=fr,German=de,Italian=it,Norwegian=nb,Brazilian Portuguese=pt,Iberian Portuguese=pt_PT,Spanish=es,Swedish=sv'
+````
+
+#### Example: `spellchecker_languages`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinymcespellchecker',
+  spellchecker_languages: 'US English=en_us,UK English=en_gb,Danish=da,Dutch=nl,Finnish=fi,French=fr,German=de,Italian=it,Norwegian=nb,Brazilian Portuguese=pt,Iberian Portuguese=pt_PT,Spanish=es,Swedish=sv'
+});
+```
+
+### `spellchecker_on_load`
+
+> **Note**: Removed in Spell Checker Pro 2.0 (TinyMCE 5.2). Spell Checker Pro will now always run on editor initialization. To disable Spell Checker Pro on load, use [`spellchecker_active`](#spellchecker_active).
+
+This option runs the spellchecker when the contents of the editor is loaded.
 
 **Type:** `Boolean`
 
 **Default Value:** `false`
 
 **Possible Values:** `true`, `false`
 
-#### Example: `spellchecker_dialog`
+#### Example: `spellchecker_on_load`
 
 ```js
 tinymce.init({
   selector: 'textarea',
   plugins: 'tinymcespellchecker',
-  toolbar: 'spellchecker',
-  spellchecker_dialog: true
+  spellchecker_on_load: true
 });
 ```
 
-### `spellchecker_whitelist`
+### `spellchecker_rpc_url`
 
-This option specifies an array of words to be ignored by the spell checker.
+This option specifies the URL of the server-side `ephox-spelling` service. For instructions on how to setup a Spell Checker Pro server-side component, see: the [server-side component installation guide]({{site.baseurl}}/enterprise/server/).
```

Comment:
"setup" to "set up" in this case maybe? I'm never quite sure with this one

Reply:
correct. "set up" fixed